lib/screens/main_screen/main_screen.dart

```diff
@@ -2,72 +2,95 @@ import 'package:carousel_slider/carousel_slider.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:go_router/go_router.dart';
-import '../../utils/department_enum.dart';
 import '../../utils/image_assets.dart';
 import '../../utils/launch_url.dart';
 import '../../utils/themes.dart';
 import '../../widgets/custom_scaffold.dart';
+import 'widget/custom_card.dart';
+import 'package:date_picker_timeline/date_picker_timeline.dart';
 
 class MainScreen extends StatelessWidget {
-  const MainScreen({Key? key}) : super(key: key);
+  MainScreen({Key? key}) : super(key: key);
```

Comment:
Don't remove const.

Either make all the variables const or move them to build method.

Moving them to build won't cause any extra runs as it is already a stateless widget